### PR TITLE
Sweetalert

### PR DESCRIPTION
**Pull request by kienpham2004**
add sweetalert

### CODE REVIEW

**Comment by kienpham2004** [comment]
ready

**Comment by TDM1911** [comment]
2 commit

File: .bowerrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "directory": "public/bower_components/"
+}
```
Comment (TDM1911) [inline .bowerrc:3]: eof

File: .gitignore [hunk 2]
```diff
@@ -10,3 +10,4 @@ Homestead.json
 Homestead.yaml
 npm-debug.log
 yarn-error.log
+/public/bower_components
```
Comment (TDM1911) [inline .gitignore:13]: eof

File: resources/views/layouts/app.blade.php [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+        <title>{{ trans('trans.title') }}</title>
+
+        <link href="https://fonts.googleapis.com/css2?family=Nunito:wght@200;600&display=swap" rel="stylesheet">
+        
+        <link rel="stylesheet" href="{{ asset('css/app.css') }}">
+    </head>
+    <body>
+    @include('sweetalert::alert')
+    @yield('content')
+    </body>
```
Comment (TDM1911) [inline resources/views/layouts/app.blade.php:20]:
```suggestion
    <body>
        @include('sweetalert::alert')
        @yield('content')
    </body>
```

**Review by huyenttm-0901 — COMMENTED** [review]
2 commits

**Pull request closed without merging**